tests/scripts/arm64_post_build.py

```diff
@@ -280,10 +323,10 @@ def main(args):
            build_type, 
            scenario, 
            core_root, 
-           g_test_url]
+           test_location]
 
    log(" ".join(args))
-   proc = subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+   proc = subprocess.Popen(args)
    std_out, std_err = proc.communicate()
```

Comment:
Nit: if you're not using `subprocess.PIPE`, std{out,err} will print to the parent's std{out,err}, so I think the call above can just be `subprocess.call()` and you can remove the code below that deals with std{out,err}.

On the other hand, it looks like you need stdout... I don't think that the changes here will actually capture it, will they?